.circleci/config.yml

```diff
@@ -356,12 +405,26 @@ commands:
                 root: ~/
                 paths:
                   - code-coverage
+      - run:
+          name: Gather system logs
+          when: always
+          command: |
+            mkdir -p ~/build-results/system-logs
+            (dmesg || :) > ~/build-results/system-logs/dmesg 2>&1
```

Comment:
I am always keen on learning new things. What does `(dmesg || :)` do?

Reply:
`:` is a noop in shell, it's just to make sure if `dmesg` for some reason fails, it doesn't cause the run to error out